test_cases/test_rooms.py

```diff
@@ -23,4 +23,11 @@ def test_livingspace_size(self):
 	def test_livingspace_type(self):
 		self.assertEqual(self.living_space.room_type, "LIVING SPACE",
 			msg = "Room Created should be of type LIVING SPACE")
-	
\ No newline at end of file
+	
+	def test_allowed_maximum_number_of_occupants_per_office(self):
+		self.assertEqual(len(self.office_space.occupants), 6,
+			msg = 'You cannot assign more than 6 people to an office')
+
+	def test_allowed_maximum_number_of_occupants_per_livingspace(self):
+		self.assertEqual(len(self.living_space.occupants), 4,
+			msg = 'You cannot assign more than 6 people to a living space')
```

Comment:
minor typo in message. 'You cannot assign more than 6 people to a living space'-- should be 4